releasetool.yaml

```diff
@@ -0,0 +1,88 @@
+---
+projects:
```

Comment:
We can discuss where to track this particular comment as well.

I discussed a bit with @pcreech that our packaging inventory is getting long in the tooth and hard to know what you are looking at. I imagine the same will end up true for this tool as well. To that end, I was thinking we should explore a directory based inventory or dynamic inventory so that package groups and releases could be split out into individual files inside a directory for easier editing and understanding.

Reply:
not sure we will exceed this here that fast, but we surely can explore that (in the future, haha).

no, really, the only thing that needs to be done for that is to point the inventory at a folder when configuring `obsah`, that should be pretty easy.

on a (slightly) related note: I am not sure where this file should actually live, in this repo or somewhere else (theforeman-rel-eng?), so that this repo only contains a tool, not the foreman specific config.